server/views/components/key-info-items/template.njk

```diff
@@ -0,0 +1,35 @@
+{% for item in params.data %}
+  <a data-featured-tile-id="{{item.id}}" 
```

Comment:
The outer CSS class for this component sits outside the component.
The `<div class=".key-info-item">` should really be in this template, if we want the CSS to be linked with this template.

Reply:
this is completely valid and I agree.

Reply:
as below